frame/staking/src/mock.rs

```diff
@@ -950,11 +958,11 @@ pub(crate) fn prepare_submission_with(
 			Staking::slashable_balance_of_fn(),
 		);
 
-		let support_map = build_support_map::<AccountId>(
+		let support_map = to_supports(
 			winners.as_slice(),
 			staked.as_slice(),
 		).unwrap();
-		evaluate_support::<AccountId>(&support_map)
+		support_map.evaluate()
```

Comment:
how can part 1 even compile ?

Reply:
not sure what you mean.

Reply:
ahh yeah part 1 was missing these, will fix there.